manifests/helm/README.md

```
@@ -0,0 +1,57 @@
+
+```bash
+# Install the chart
+helm install cerberus-mergeguard oci://ghcr.io/heathcliff26/manifests/cerberus-mergeguard --version <version>
+```
+
+## Configuration
+
+### Minimal Configuration (No Ingress)
+
+You need to provide a secret for the github app's private key.
+```yaml
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)